.github/workflows/build.yml

```
@@ -0,0 +1,43 @@
+on: push
```

Should we also build on pull_request, since travis also builds both?
```suggestion
on: [push, pull_request]
```

---

Had a quick look here:

* https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request
https://help.github.com/en/actions/reference/events-that-trigger-workflows#push-event-push

I don't think we need a separate trigger for PRs, push should be sufficient as we have a build for this PR as well?

---

I think there is a difference between Travis PR builds and GitHub. Travis will always built the "merged" commit https://docs.travis-ci.com/user/pull-requests/#how-pull-requests-are-built. But as far as I understood, the `pull_request` on GitHub trigger will only built the latest commit (thus essentially the same as push).
And since our workflow keeps the PR up to date, this should be no issue.

---

> We need this for PRs from forked repos.

right ...